src/sampleutil.cpp

```diff
@@ -296,26 +296,31 @@ void SampleUtil::convertS16ToFloat32(CSAMPLE* _RESTRICT pDest, const SAMPLE* _RE
 }
 
 // static
-bool SampleUtil::sumAbsPerChannel(CSAMPLE* pfAbsL, CSAMPLE* pfAbsR,
+SampleUtil::CLIPPING SampleUtil::sumAbsPerChannel(CSAMPLE* pfAbsL, CSAMPLE* pfAbsR,
         const CSAMPLE* pBuffer, int iNumSamples) {
     CSAMPLE fAbsL = CSAMPLE_ZERO;
     CSAMPLE fAbsR = CSAMPLE_ZERO;
-    CSAMPLE clipped = 0;
+    CSAMPLE clippedL = 0;
+    CSAMPLE clippedR = 0;
```

Comment:
I highly recommend using an IDE. They make working on a huge project spread across hundreds of files much easier.